src/win/win_impl.rs

```diff
@@ -151,10 +151,20 @@ impl Mouse for Enigo {
 
     fn move_mouse(&mut self, x: i32, y: i32, coordinate: Coordinate) -> InputResult<()> {
         debug!("\x1b[93mmove_mouse(x: {x:?}, y: {y:?}, coordinate:{coordinate:?})\x1b[0m");
-        let flags = if coordinate == Coordinate::Abs {
-            MOUSEEVENTF_MOVE | MOUSEEVENTF_ABSOLUTE
+        let (flags, x, y) = if coordinate == Coordinate::Abs {
+            // 0-screen width/height - 1 map to 0-65535
+            // Add w/2 or h/2 to round off
+            // See https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-mouse_event#remarks
+            let (w, h) = self.main_display()?;
+            let w = w as i64 - 1;
+            let h = h as i64 - 1;
+            let x = x as i64;
+            let y = y as i64;
+            let x = (x * 65535 + w / 2 * x.signum()) / w;
```

Comment:
What is the `+ w / 2 * x.signum()` for? If I want to move the mouse to 0, the result should still be zero, right? With this, will the mouse not be moved to the center of the screen? The coordinate system of `enigo` starts in the top left corner.

I think this should be:

```
let x = (x * 65535) / w;
let y = (y * 65535) / h;
```

Reply:
`let x = (x * 65535 + w / 2 * x.signum()) / w;` = `let x = ((x as f64 * 65535.0) / w as f64).round() as i64;`
This is a technique for rounding using integers. It addresses potential precision loss issues when converting integers to floating-point numbers, hence employing this method.